### DISCUSSION TITLE
nginx反代后404

### DISCUSSION THREAD

**Discussion by firesky123** [AlistGo/alist#345] [Q&A] [answered]
按照文档里注释掉这三个后
location ~ ^/(.user.ini|.htaccess|.git|.svn|.project|LICENSE|README.md
location ~ .*.(gif|jpg|jpeg|png|bmp|swf)$
location ~ .*.(js|css)?$
访问网站提示
Error
Request failed with status code 404

**Comment by xhofe** [comment, marked as answer]
非本程序问题，无法解答。

**Reply by firesky123** [reply]
发现了问题所在，反代到 / 下就可以访问，如果加了目录反代就404，比如 mydomain.com/download 或者 mydomain.com/download/ 就会提示Error Request failed with status code 404

**Reply by xhofe** [reply]
- #370

**Reply by yigedabuliu** [reply]
找到你 Alist 目录下 /data/config.json 中的 site_url 必须与[反向代理]后的访问地址一致，否则会生成错误的 API 路径
"site_url": "http://127.0.0.1:5244[你的域名或ip]/alist[你的目录]",
